app/static/src/js/headscripts/gtm.js

```diff
@@ -23,6 +23,44 @@ function add_GTM() {
     GTM_Loaded = true;
 }
 
+function push_to_datalayer(event, category_code = null, category_name = null, category_traversal = null, diagnosis_result = null) {
+    window.dataLayer = window.dataLayer || [];
```

Comment:
If this won't be generic then probably move this back to the main window load bit so it covers the lot.